cypress/e2e/login.cy.js

```diff
@@ -0,0 +1,19 @@
+describe("logging in to page with email and password", () => {
+  it("passes", () => {
+    cy.visit("https://martinsn676.github.io/social-media-client/");
```

Comment:
```suggestion
    cy.visit("/");
```

When we use cypress, it's important to test the files in your PR - rather than the files that are already deployed to the server. Cypress launches its own local server, so you don't need to specify the domain name.